platforms/network-schema.json

```diff
@@ -970,7 +970,7 @@
       "type": "object",
       "properties": {
         "keystore": {
-          "type": "object",
```

Comment:
kindly review the schema, for the credentials. doesn't match with current network.yaml structure